bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BByteArray.java

```diff
@@ -55,6 +55,11 @@ public byte get(long index) {
         return values[(int) index];
     }
 
+    public byte[] getBytes() {
+        BByteArray copy = (BByteArray) this.copy();
+        return copy.values;
```

Comment:
Can't we simply return `values.clone`?

Reply:
Yes, since this is a primitive array, we can use it.